clang/lib/CIR/Dialect/Transforms/LoweringPrepare.cpp

```diff
@@ -1061,6 +1077,25 @@ void LoweringPreparePass::lowerIterEndOp(IterEndOp op) {
   op.erase();
 }
 
+void LoweringPreparePass::buildGlobalAnnotationValues() {
+  if (globalAnnotations.empty())
+    return;
+  std::vector<mlir::Attribute> annotationValueVec;
```

Comment:
SmallVector always should be used, if you are using `std::vector` please provide a good explanation here or in any follow up PR. You can get more used to LLVM idioms here: https://llvm.org/docs/ProgrammersManual.html